pkg/compiler/types.go

```diff
@@ -645,6 +656,16 @@ var typeString = &typeDesc{
 				NoZ:        t.Ident == stringnoz,
 			}
 		}
+		if len(args) > 0 && t.Ident == glob {
+			base.TypeSize = 0
+			return &prog.BufferType{
+				TypeCommon: base.TypeCommon,
+				Kind:       prog.BufferGlob,
+				SubKind:    args[0].String,
+				Values:     make([]string, 1),
```

Comment:
Do we need to pass this string slice of size 1 with an empty string as first element? Why? Can we drop this?